### PR TITLE
Dont panic345 patch 1

### PR DESCRIPTION
**Pull request by DontPanic345**
Created as a fix to

apm is not installed on Windows, no "Install Shell Commands" option #381

Added info about setting the path environment variable on Windows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,10 +26,17 @@ Atom packages.
...
 ## Installing
 
-apm is bundled and installed automatically with Atom. You can run the
-_Atom > Install Shell Commands_ menu option to install it again if you aren't
+apm is bundled and installed automatically with Atom.
+
+On a Mac you can run the _Atom > Install Shell Commands_ menu option to install it again if you aren't
 able to run it from a terminal.
 
+On Windows you will need to add it too your path enviroment variable youself, from commandline run
```
Comment (kevinsawicki) [inline README.md]: The Windows installer should do this automatically for you.

**Comment by bronson** [comment]
@DontPanic345 just curious, what's the status of this PR?  Does the problem (Win installer not setting path) still exist?

**Comment by 50Wliu** [comment]
> just curious, what's the status of this PR? Does the problem (Win installer not setting path) still exist?

It worked when I (and most others) installed Atom - it sounds like something went wrong with @DontPanic345's installation.

**Comment by damieng** [comment]
There were some issues with Atom not setting the path if registry tools had been disabled or there were unicode characters in the current path.  Both were addressed in thttps://github.com/atom/atom/pull/10326 hat will be released with Atom 1.7.0

**Pull request closed without merging**